.mergify.yml

```diff
@@ -20,7 +20,7 @@ pull_request_rules:
           {{ body }}
     conditions:
       - -title~=(WIP|wip)
-      - -label~=(do-not-merge)
+      - -label~=(🚧 pr/do-not-merge|⚠️ pr/review-mutation)
```

Comment:
Make sure to check in the config editor to see if it picks these up properly. Mostly concerned about the `/`

Reply:
No worries, I did thoroughly check first before commiting.